src/main/java/com/pojos/User.java

```diff
@@ -6,5 +6,37 @@
 	private String emailId;
 	private String mobileNumber;

```

Comment:
It is better to change String to Integer for mobileNumber

Reply:
Ok I'll change as for your requirement